docker/supabase/volumes/db/init/05-github-schema.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE IF NOT EXISTS github_users (
+  id serial PRIMARY KEY,
+  login text, -- the user name of the user, known as the `login` field in GitHub API responses
+  type text check ( type IN ('user', 'organization') ) NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS github_repositories (
+  id serial PRIMARY KEY,
+  repo_name text,
```

Comment:
NOT NULL? A repo without a name doesn't make sense to me.

Reply:
So, I added NOT NULL to everything (a repo cannot exist without an owner, and an issue cannot exist without belonging to a repo).